### PR TITLE
fix search

### PR DESCRIPTION
**Pull request by DmytroHryshyn**
-fixes incorrect search behaviour 
https://app.claap.io/c-BT2DRbCjzO-ZnMKhzBEt2xj?ccl=1

### CODE REVIEW

File: intuita-webview/src/fileExplorer/TreeView/index.tsx [hunk 1]
```diff
@@ -77,7 +77,11 @@ const TreeView = ({
 	stagedJobs,
 }: Props) => {
 	const allFileNodesReady = fileNodes !== null;
+
+	// @TODO  @UX Need to show info message to users.
```
Comment (grzpab) [inline intuita-webview/src/fileExplorer/TreeView/index.tsx:81]: @DmytroHryshyn is there a ticket for it?

File: intuita-webview/src/fileExplorer/TreeView/index.tsx [hunk 2]
```diff
@@ -223,22 +227,22 @@ const TreeView = ({
 		}
 	}, [focusedNodeId, fileNodeIds]);
 
+	if (fileNodes === null || fileNodes.length === 0) {
```
Comment (grzpab) [inline intuita-webview/src/fileExplorer/TreeView/index.tsx:230]: how come `fileNodes` are nullable?

File: intuita-webview/src/fileExplorer/TreeView/index.tsx (hunk 1 above)
Comment (DmytroHryshyn) [reply intuita-webview/src/fileExplorer/TreeView/index.tsx:81]: i will add a ticket

File: intuita-webview/src/fileExplorer/TreeView/index.tsx (hunk 2 above)
Comment (DmytroHryshyn) [reply intuita-webview/src/fileExplorer/TreeView/index.tsx:230]: i will update fileNodes to be an array

**Review by grzpab — APPROVED** [review]

**Pull request merged**